setup.py

```diff
@@ -27,6 +27,14 @@
         'ethereum==2.3.0',
         'web3==4.8.2',
         'rlp==0.6.0',
-        'py-solc-simple==0.0.13'
-    ]
+        'py-solc-simple==0.0.14',
+        'eip712-structs==1.1.0'
```

Comment:
curious, why we don't need this before?

Reply:
We did, but previously it was mentioned only in the requirements.txt file